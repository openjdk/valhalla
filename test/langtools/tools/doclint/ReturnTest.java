/*
 * @test /nodynamiccopyright/
 * @modules jdk.javadoc/jdk.javadoc.internal.doclint
 * @build DocLintTester
 * @run main DocLintTester -ref ReturnTest.out ReturnTest.java
 */

/** No comment. */
public class ReturnTest {
    /**
     {@return legal} **/
    public int m_legal() { return 0; }

    /** <p> {@return illegal} **/
    public int m_illegal_html() { return 0; }

    /** text {@return illegal} **/
    public int m_illegal_text() { return 0; }

    /** &amp;{@return illegal} **/
    public int m_illegal_entity() { return 0; }

    /** @@{@return illegal} **/
    public int m_illegal_escape() { return 0; }

    /** {@return legal} text {@return illegal} **/
    public int m_illegal_repeat() { return 0; }

    /** . */
    private ReturnTest() { }
}
