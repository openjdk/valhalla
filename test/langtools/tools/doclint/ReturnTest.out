ReturnTest.java:14: warning: {@return} not at beginning of description
    /** <p> {@return illegal} **/
            ^
ReturnTest.java:17: warning: {@return} not at beginning of description
    /** text {@return illegal} **/
             ^
ReturnTest.java:20: warning: {@return} not at beginning of description
    /** &amp;{@return illegal} **/
             ^
ReturnTest.java:23: warning: {@return} not at beginning of description
    /** @@{@return illegal} **/
          ^
ReturnTest.java:26: warning: @return has already been specified
    /** {@return legal} text {@return illegal} **/
                             ^
ReturnTest.java:26: warning: {@return} not at beginning of description
    /** {@return legal} text {@return illegal} **/
                             ^
6 warnings